File: Conferences/ClientApp/src/app/models/Section.ts
import { Conference } from '../models/Conference';
import { EventObject } from './Event';

export class Section {
  id: number | null = null
  name: string = ''
  startDate: Date  | null = null
  endDate: Date | null = null
  Conference: Conference | null = null
  ConferenceId: number | null = null
  Events: EventObject[] | null = null
}

File: Conferences.Domain/Entities/Section.cs
using Microsoft.EntityFrameworkCore;

namespace Conferences.Domain.Entities;
public class Section
{
    public int? Id { get; set; }

    public string Name { get; set; } = string.Empty;

    public DateTime StartDate { get; set; }

    public DateTime EndDate { get; set; }

    public Conference? Conference { get; set; }

    public int ConferenceId { get; set; }

    public List<Event>? Events { get; set; } = new List<Event>();
}
